src/60_AMP_Ads/Video_Ad.html

```diff
@@ -0,0 +1,197 @@
+
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+  <!-- ## Styling -->
+  <!--
+  This is a relatively advanced example that requires some styling to make it
+  look and function properly.
+  -->
+  <style amp-custom>
+    #splash-container, #video-container {
+      /* Set to ad dimensions */
+      width: 300px;
+      height: 168px;
+
+      /* Set so children can position themselves in relation to the container */
+      position: relative;
+
+      /* Basic font styles */
+      font-family: sans-serif;
+      line-height: 1;
+      color: #fff;
+    }
+    .brand {
+      /* Place in top-left corner */
+      position: absolute;
+      z-index: 1;
+      top: 10px;
+      left: 10px;
+    }
+    .watch-video {
+      /* Position roughly in vertical center */
+      position: absolute;
+      z-index: 1;
+      top: 70px;
+      left: 10px;
+
+      /* Button styles */
+      padding: 7px;
+      border: gray solid 1px;
+      border-radius: 7px;
+      background-color: rgba(0, 0, 0, 0.8);
+      cursor: pointer;
+    }
+    .title {
+      /* Place beneath "Watch Video" button */
+      position: absolute;
+      z-index: 1;
+      top: 110px;
+      left: 10px;
+      text-shadow: 2px 2px 8px rgba(0, 0, 0, 0.8);
+
+      /* Font styles */
+      font-weight: bold;
+      font-size: 18px;
+    }
+    .close {
+      /* Place in top-right corner */
+      position: absolute;
+      z-index: 1;
+      right: 10px;
+      top: 10px;
+
+      /* Button styles */
+      cursor: pointer;
+      font-weight: bold;
+      font-size: 18px;
+      width: 24px;
+      height: 24px;
+      line-height: 24px;
+      text-align: center;
+    }
+    #pause, #play {
+      /* Place in bottom-right corner */
+      position: absolute;
+      z-index: 1;
+      right: 10px;
+      bottom: 10px;
+
+      /* Button styles */
+      border: 1px solid grey;
+      vertical-align: center;
+      text-align: center;
+      background-color: rgba(0,0,0, 0.8);
+      cursor: pointer;
+      width: 32px;
+      height: 32px;
+      line-height: 32px;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- ## Full Markup -->
+  <!--
+  Create the necessary markup to show two layers that represent the two ad
+  states.
+
+  **Splash layer**
```

Comment:
I actually like it - it demonstrates the behavior of each section in isolation. -> You get the overall behavior in the demo view. You can make this more explicit by saying something like: 'the video layer is initially hidden, but will appear when you click 'watch video' above.'. 

I think it'd be nice to have the sample then split into four sections:

* Introduction
* Setup
* Splash Layer
* Video Layer

But I'm happy either way - pick whatever you think works best. It's your sample 😉.